dev-docs/modules/userid-submodules/freepass.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: userid
+title: FreePass ID
+description: FreePass User ID sub-module
+useridmodule: freepassIdSystem
+---
+
+[FreePass](https://freepass-login.com/introduction.html) is a common authentication service operated by Freebit Co., Ltd. Users with a FreePass account do not need to create a new account to use partner services.
+
+Please contact FreePass before using this ID.
```

Comment:
@czarpino - are you assuming that people will contact FreePass via the web link above? Some modules provide an email address...

Reply:
Thank you for pointing it out! I've added the contact email. 🙏